lib/debride.rb

```diff
@@ -259,6 +259,22 @@ def process_call sexp # :nodoc:
       new_name = sexp[3]
       new_name = new_name.last if Sexp === new_name # when is this NOT the case?
       known[new_name] << klass_name if option[:rails]
+    when :attr_accessor then
+      sexp[3..-1].each do |new_name|
```

Comment:
These aren't warning free. Are you running the tests with rake? It should be outputting warnings.